editions/tw5.com/tiddlers/system/tw5.com-styles.tid

```diff
@@ -197,3 +198,24 @@ div.content {
 .tc-btn-download:active {
 	background: #1475ff;
 }
+
+/* WikiText rules */
+
+.multi-columns,
+.fourcolumns {
+  display: block;
+  column-count: 4;
+  column-gap: 1em;
+}
+
+/* make `code` elements less distracting */
```

Comment:
This sounds like a somewhat opinionated visual change. In such situations it is much easier to review the PR if you can include before and after screenshots.

Also please note that our coding style is to use sentence case comments (ie the first letter should be capitalised).